scripts/hdiff.sh

```diff
@@ -0,0 +1,8 @@
+base=$(basename $1 .test)
+
+out=$(find . -name ${base}.out)
+exp=$(find . -name ${base}.regexpect)
+
+ORION_DIFF=${ORION_DIFF:-tkdiff}
+
+$ORION_DIFF $out $exp
```

Comment:
Removing hdiff.sh and modifying viewDiff.sh.

The problem with CB_DIFF_TOOL is solved with this alias (just for me):

    alias hdiff='CB_DIFF_TOOL=tkdiff viewDiffs.sh'